### DISCUSSION TITLE
west flash to teensy 41 fail, use blinky with west build

### DISCUSSION THREAD

**Discussion by WolsYang** [zephyrproject-rtos/zephyr#37339] [Q&A]
I follow the Getting Started Guide, then
1. zephyrproject\zephyr>$ west build -p auto -b teensy41 samples\basic\blinky
2. zephyrproject\zephyr>$ west flash   <<Fail
3. zephyrproject\zephyr>$ cd build
4. zephyrproject\zephyr\build>$ west flash   <<Fail
5.  zephyrproject\zephyr\build>$ ninja flash   <<Fail
Here is the error message show in windows10 cmd :
[0/1] Flashing teensy41
-- west flash: rebuilding
ninja: no work to do.
FATAL ERROR: either a pristine build is needed, or board teensy41 doesn't support west flash/debug (no ZEPHYR_RUNNERS_YAML in CMake cache)
FAILED: zephyr/cmake/flash/CMakeFiles/flash C:/Users/canic/Desktop/zephyrproject/zephyr/build/zephyr/cmake/flash/CMakeFiles/flash
cmd.exe /C "cd /D C:\Users\canic\Desktop\zephyrproject\zephyr\build && "C:\Program Files\CMake\bin\cmake.exe" -DTARGET=flash -DDEPENDENCIES="" -P C:/Users/canic/Desktop/zephyrproject/zephyr/cmake/flash/check_runner_dependencies.cmake && "C:\Program Files\CMake\bin\cmake.exe" -E env C:/Users/canic/AppData/Local/Programs/Python/Python39/python.exe -m west flash"
ninja: build stopped: subcommand failed.

Is any step I do it wrong?

**Comment by galak** [comment]
Nothing wrong, the board doesn't support west flash/debug.  There is no `board.cmake` describing how to invoke the west runners for flashing purposes.

**Reply by WolsYang** [reply]
So although teensy4 show on suport supported boards list and zephyrproject\zephyr\boards\arm\teensy4, I still need to lear how to make booard.make file, right?

**Comment by galak** [comment]
Converted to a Q&A discussion as this is not a bug.
